modules/videoio/test/test_mfx.cpp

```diff
@@ -6,6 +6,19 @@
 
 namespace opencv_test { namespace {
 
+TEST(videoio_mfx, yuv_frame)
+{
+    VideoCapture cap;
+    ASSERT_NO_THROW(cap.open(0, CAP_MSMF));
```

Comment:
Tests which require camera device should go to https://github.com/opencv/opencv/blob/4.x/modules/videoio/test/test_camera.cpp